.github/workflows/send-redirects-to-staging.yml

```diff
@@ -0,0 +1,57 @@
+
+# Terraform Action to lint and apply updated redirects
+name: AWS Cloud Database Test Execution
+on:
+  push:
+    branches:
+      - master
+      - main
+      - github-action-*
```

Comment:
is this required?

Reply:
I wanted to make the action execute automatically (for staging only) when redirects are updated. The `github-action-*` part lets us test workflow changes in a test branch